### ISSUE TITLE
Can't make the CloudClipBoard connect under android

### ISSUE THREAD

**Issue by GP-ES** [SchneiderInfosystems/FB4D#34]
Hi,

Thanks for your work! it's a piece of art very useful for the Delphi community.
I've started testing the CloudClipBoard demo. I've created a test Firebase project and It works as expected under windows 10.
I've  registered the project in the Google Firebase console and also registered the CloudClipBoard FMX Android application. I've set the package name and  the App SHA-1.
But still CloudClipBoard demo is not able to connect to the firebase Authentication under Android 9. 
It gets waiting on the "please wait for firebase" Screen forever.
I'm running Delphi 10.2 with Update 3. Is there any document that describes the steps to make the app run under Android (Settings for Firebase, for the app, etc.)
Thanks a lot!

**Comment by SchneiderInfosystems** [comment]
The CloudClipBoard application works properly on my different Android Devices (running on Android 9 and 8). Unfortunately, I don't have prepared a youtube video on my channel, that shows this.

Of course, the app needs internet permission and also access the firebase cloud. As first the URL `https:/www.googleapis.com(indetitytoolkit/v3/relingpart` will be accessed. You can check if you can reach this website in a browser on the concerned device.

Alternatively, you can set a breakpoint at start of the `FB4D.Request.TFirebaseRequest.SendRequest` to see what happens with the first cloud request. 

At the beginning of my work, I used Delphi 10.2.3 but I recommend to use currently 10.3.2 or 10.3.3.

Hope this helps you to identify the root cause. Please inform us of your findings.

**Comment by GP-ES** [comment]
Hi,

After testing with 10.3.2 community and having the same result, I've decided to create a new google account (I was using my personal google account for testing before) .
Now I've used my organizational account and now everything works under 10.2.3 and  10.3.2  over Android 9.
I really can't tell why my personal account doesn't as everything is set up exactly the same way between both accounts under firebase but the problem was there!
Thanks for your support.
